src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -788,8 +788,10 @@ void GrpcLb::BalancerCallState::Orphan() {
 void GrpcLb::BalancerCallState::StartQuery() {
   GPR_ASSERT(lb_call_ != nullptr);
   if (GRPC_TRACE_FLAG_ENABLED(grpc_lb_glb_trace)) {
-    gpr_log(GPR_INFO, "[grpclb %p] lb_calld=%p: Starting LB call %p",
-            grpclb_policy_.get(), this, lb_call_);
+    char* peer_string = grpc_call_get_peer(lb_call_);
+    gpr_log(GPR_INFO, "[grpclb %p] peer=%s lb_calld=%p: Starting LB call %p",
```

Comment:
I don't think it's going to work at this point, because the peer isn't known until after the send_initial_metadata op completes, since that's what triggers the LB pick.

I think this needs to be logged in `OnInitialRequestSentLocked()`.